editor/src/components/inspector/common/css-utils.ts

```diff
@@ -4746,3 +4749,163 @@ const LayoutPropertyList = [
 export function isLayoutPropDetectedInCSS(cssProps: { [key: string]: any }): boolean {
   return LayoutPropertyList.findIndex((prop: string) => cssProps[prop] != null) > -1
 }
+
+interface NonTrivialKeyword {
+  trivial: false
+}
+
+const nontrivial: NonTrivialKeyword = { trivial: false }
+
+type ParsedPropertiesWithNonTrivial = {
+  [Property in keyof ParsedProperties]: ParsedProperties[Property] | NonTrivialKeyword
+}
+
+export const trivialDefaultValues: ParsedPropertiesWithNonTrivial = {
```

Comment:
@maltenuhn  this dictionary will need some QA. for every trivial value we want to hide (like maxWidth: none) the entry here must match the value found in `computedStyle`. 

We can't use the default useragent stylesheet because at least in Chrome, the length units for example are computed as `0px` instead of `0`

Reply:
I've taken a look through, and will leave a couple of annotations, but overall this looks good to me and we can merge it as is. (I take it this dictionary implies that every property can have zero or one values we can hide?)

Reply:
> I've taken a look through, and will leave a couple of annotations, but overall this looks good to me and we can merge it as is. 

> (I take it this dictionary implies that every property can have zero or one values we can hide?)

yes. if that is not good enough, we can change it later to some other format

Reply:
> yes. if that is not good enough, we can change it later to some other format

Nah,  fine as is. If we truly have multiple values that could each be meaningless, that fact in itself probably makes them meaningful :)